.travis.yml

```diff
@@ -31,7 +31,7 @@ before_script:
 
 script:
   - bundle exec rspec spec -fd
-  - bundle exec rubocop app spec -R --format simple
+ # - bundle exec rubocop app spec -R --format simple
```

Comment:
hum, no! I think it is a bad idea. If you have some error with rubocop on travis, please fix them. But do not disable the test ;)

Reply:
That was a mistake, I don't know why I commented that.